position/_RPC_dry.gno

```diff
@@ -33,38 +42,34 @@ func DryMint(
 	i256Liquidity := i256.FromUint256(liquidity)
 
 	// FROM: pool__position_modify.gno
-	var amount0, amount1 *i256.Int
-	if !(liquidity.IsZero()) { // != 0
-		if tickCurrent < tickLower {
-			amount0Str := common.SqrtPriceMathGetAmount0DeltaStr(
-				sqrtLowerX96,
-				sqrtUpperX96,
-				i256Liquidity,
-			)
-			amount0 = i256.MustFromDecimal(amount0Str)
-		} else if tickCurrent < tickUpper {
-			amount0Str := common.SqrtPriceMathGetAmount0DeltaStr(
-				sqrtRatioX96,
-				sqrtUpperX96,
-				i256Liquidity,
-			)
-			amount0 = i256.MustFromDecimal(amount0Str)
+	amount0Str := common.SqrtPriceMathGetAmount0DeltaStr(
+		sqrtLowerX96,
+		sqrtUpperX96,
+		i256Liquidity,
+	)
+
+	amount1Str := common.SqrtPriceMathGetAmount1DeltaStr(
+		sqrtLowerX96,
+		sqrtRatioX96,
+		i256Liquidity,
+	)
 
-			amount1Str := common.SqrtPriceMathGetAmount1DeltaStr(
-				sqrtLowerX96,
-				sqrtRatioX96,
-				i256Liquidity,
-			)
-			amount1 = i256.MustFromDecimal(amount1Str)
-		} else {
-			amount1Str := common.SqrtPriceMathGetAmount1DeltaStr(
-				sqrtLowerX96,
-				sqrtUpperX96,
-				i256Liquidity,
-			)
-			amount1 = i256.MustFromDecimal(amount1Str)
-		}
+	var amount0, amount1 *i256.Int
+	switch {
+	case liquidity.IsZero():
+		amount0 = i256.Zero()
```

Comment:
73 ~ 74 line initialize nil to zero, therefor we can remove `case liquidity.IsZero()` logic

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/296/commits/299d6f8bd76599de0ad674ee87784fc7ccfe8f9f